packages/lesswrong/lib/tableOfContents.ts

```diff
@@ -1,3 +1,9 @@
+import { commentsTableOfContentsEnabled } from "./betas";
+import * as _ from 'underscore';
+import { answerTocExcerptFromHTML, truncate } from "./editor/ellipsize";
+import { htmlToTextDefault } from "./htmlToText";
+import type { WindowType } from "./domParser";
```

Comment:
A lot of this file is moved from `packages/lesswrong/server/tableOfContents.ts`, although most of the functions have been rewritten